### PR TITLE
UI issues on product pages

### LINKED ISSUE #937: UI issues on product pages
1. Please reduce the size of these images. Also in the URL  it should be ".com" instead of the hyphen 
https://franklin.moleculardevices.com/products/3d-biology/3d-bioprinting-bab400-bioprinter-automation-solution-com

image

2. Remove the line separator above "Assure data integrity.." 
https://franklin.moleculardevices.com/products/microplate-readers/absorbance-readers/spectramax-abs-plate-readers
https://franklin.moleculardevices.com/products/microplate-readers/acquisition-and-analysis-software/softmax-pro-software
https://franklin.moleculardevices.com/products/gxp-compliance-solutions/softmax-pro-gxp-software-installation-validation-services

3. Add a little space here
https://franklin.moleculardevices.com/products/microplate-readers/multi-mode-readers/filtermax-f3-f5-readers

image

4. Centre align "Cellular Image Gallery" text.  Add space above the disclaimer text and remove the letter "S" and remove the extra asterisks. 
https://franklin.moleculardevices.com/products/cellular-imaging-systems/high-content-imaging/imagexpress-confocal-ht-ai 
https://franklin.moleculardevices.com/products/cellular-imaging-systems/high-content-imaging/imagexpress-micro-confocal

image

5. Fix the font size for this section- 
**Center align "Turnkey solutions for ..."  with font h2:36 px.  
"MetaXpress Application Modules"  h2: 24px blue color 
"Expedite image analysis and data...." h4:18px**
 https://franklin.moleculardevices.com/products/cellular-imaging-systems/acquisition-and-analysis-software/metaxpress

image

6. View Interactive Demo CTA is not working, and remove the cross button below. Add line separators here 
 https://franklin.moleculardevices.com/products/cellular-imaging-systems/acquisition-and-analysis-software/cellreporterxpress

image

7. Add line separator 
https://franklin.moleculardevices.com/products/clone-screening/single-cell-isolation/dispencell-single-cell-dispenser

image

8. Remove Request pricing CTA from the banner 
https://franklin.moleculardevices.com/products/flipr-penta-high-throughput-cellular-screening-system

9. Empty resources
https://franklin.moleculardevices.com/service-support/lab-automation-solutions/lab-automation-for-high-content-screening#resources

10. Remove line separator 
https://franklin.moleculardevices.com/products/gxp-compliance-solutions/softmax-pro-gxp-software
https://franklin.moleculardevices.com/products/gxp-compliance-solutions/softmax-pro-gxp-software-installation-validation-services

image

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #937

Test URLs:

Fixes done:
1. Please reduce the size of these images
https://franklin.moleculardevices.com/products/3d-biology/3d-bioprinting-bab400-bioprinter-automation-solution.com

2. Remove the line separator above "Assure data integrity.."
https://franklin.moleculardevices.com/products/microplate-readers/absorbance-readers/spectramax-abs-plate-readers
https://franklin.moleculardevices.com/products/microplate-readers/acquisition-and-analysis-software/softmax-pro-software
https://franklin.moleculardevices.com/products/gxp-compliance-solutions/softmax-pro-gxp-software-installation-validation-services

3. Add a little space here
- Before: https://main--moleculardevices--hlxsites.hlx.page/products/microplate-readers/multi-mode-readers/filtermax-f3-f5-readers
- After: https://937-products--moleculardevices--hlxsites.hlx.page/products/microplate-readers/multi-mode-readers/filtermax-f3-f5-readers

4. Centre align "Cellular Image Gallery" text. Add space above the disclaimer text and remove the letter "S" and remove the extra asterisks.
- Before: https://main--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/high-content-imaging/imagexpress-confocal-ht-ai
- After: https://937-products--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/high-content-imaging/imagexpress-confocal-ht-ai

- Before: https://main--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/high-content-imaging/imagexpress-micro-confocal
-After: https://937-products--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/high-content-imaging/imagexpress-micro-confocal

5. Fix the font size for this section-
Center align "Turnkey solutions for ..." with font h2:36 px.
"MetaXpress Application Modules" h2: 24px blue color
"Expedite image analysis and data...." h4:18px
- Before: https://main--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/acquisition-and-analysis-software/metaxpress
- After: https://937-products--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/acquisition-and-analysis-software/metaxpress

6. View Interactive Demo CTA is not working, and remove the cross button below. Add line separators here
https://franklin.moleculardevices.com/products/cellular-imaging-systems/acquisition-and-analysis-software/cellreporterxpress

7. Add line separator
https://franklin.moleculardevices.com/products/clone-screening/single-cell-isolation/dispencell-single-cell-dispenser

8. @MeghnaTewari Please clarify: Remove Request pricing CTA from the banner
The Request Pricing is on the original page on the banner too.
https://franklin.moleculardevices.com/products/flipr-penta-high-throughput-cellular-screening-system

9. Empty resources
https://franklin.moleculardevices.com/service-support/lab-automation-solutions/lab-automation-for-high-content-screening#resources

10. Remove line separator
https://franklin.moleculardevices.com/products/gxp-compliance-solutions/softmax-pro-gxp-software
https://franklin.moleculardevices.com/products/gxp-compliance-solutions/softmax-pro-gxp-software-installation-validation-services

Additional reported styling issues for Category pages (blue title, title center align, font html snippet, buttons in same row)
- Before: https://main--moleculardevices--hlxsites.hlx.page/products/gxp-compliance-solutions
- After: https://937-products--moleculardevices--hlxsites.hlx.page/products/gxp-compliance-solutions

### CODE REVIEW

**Comment by lydiapuric** [comment]
Styling issues got successfully validated by Meghna

File: fragments/products/gxp-compliance-solutions-toc.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
   .gxp-product.product-section .product-cat .product-item p {
     text-align: center;
     max-width: 100%;
-    font-family: "Proxima Nova Rg";
+    font-family: "Proxima Nova Regular";
```
Comment (mhaack) [inline fragments/products/gxp-compliance-solutions-toc.html:37]: Use font variable!

File: styles/styles.css [hunk 2]
```diff
@@ -1724,6 +1741,10 @@ main .section.background-molecules {
   color: #008da9;
 }
 
+.section.blue-subtitle div > h3 {
```
Comment (mhaack) [inline styles/styles.css:1744]: Don't we already have blue title styles for h3?

Comment (lydiapuric) [reply styles/styles.css:1744]: I needed a blue style for subtitles without touching the h2 to be blue.

File: fragments/products/gxp-compliance-solutions-toc.html (hunk 1 above)
Comment (lydiapuric) [reply fragments/products/gxp-compliance-solutions-toc.html]: Done!

File: styles/styles.css (hunk 2 above)
Comment (lydiapuric) [reply styles/styles.css:1744]: as discussed, ok like this as i need to differentiate between title and subtitle.

**Review by mhaack — APPROVED** [review]

**Pull request merged**